### ISSUE TITLE
A11y_Azure Data Studio_Toolbar_Zoom In_Zoom: At resize the tooltip present at some of the nodes are getting truncated.

### ISSUE THREAD

**Issue by Rama8345** [microsoft/azuredatastudio#20796]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #Zoom; #WCAG1.4.4; #FTP; #Regressed:11-01-22; #Closed

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:
1. Launch the "Azure data studio" application.
2. Connect with the provided "sqltools2019-3" SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. TAB to the toolbar present at right side of the screen.
7. TAB to "Zoom In" button and hit ENTER key.
8. Zoom the screen to 200% and tab to any node having long tooltip observe the issue.

### Actual:
At resize the tooltip present at some of the nodes are getting truncated.

**This issue is also observed for the Zoom Out/ Zoom to fit operations also.**

### Expected Result:
At resize also the tooltip present at any of the nodes should not truncate and should be properly visible.

### Attachment:
![A11y_Azure Data Studio_Toolbar_Zoom In_Zoom](https://user-images.githubusercontent.com/115216656/194931924-cca7e6b9-0667-44d8-9f0d-93b11e322567.png)

**Comment by aasimkhan30** [comment]
@Rama8345, this issue is fixed in the latest insiders. Can you please re test it?

**Comment by Rama8345** [comment]
@aasimkhan30 
Verified the bug in the latest insiders build, but we are getting below error.
![image](https://user-images.githubusercontent.com/115216656/199223982-4d526e37-ff58-4f98-9410-d235badfb14b.png)

**Comment by Rama8345** [comment]
Verified the issue in the latest insiders build. The issue is fixed. So closing the bug. Please find attachment for reference.
![20796_Closed](https://user-images.githubusercontent.com/115216656/199414040-d9a78751-fc6a-4ac9-b20e-6322e147cc30.gif)
